### PR TITLE
Added legacy trail page and route

### PR DESCRIPTION
**Pull request by Big-nuche**

### CODE REVIEW

File: resources/views/pages/trails/legacy.blade.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
+@extends('layouts.app')
+
+@section('title', 'About eBikes Florida')
```
Comment (Leping08) [inline resources/views/pages/trails/legacy.blade.php]: Can you update the page title and SEO for this page?

File: resources/views/pages/trails/legacy.blade.php [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+@section('description',
```
Comment (Leping08) [inline resources/views/pages/trails/legacy.blade.php:5]: Update the SEO here as well.

File: resources/views/pages/trails/legacy.blade.php [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    'Explore the benefits of electric bikes in Florida. Our site provides information and
+    resources.')
+
+@section('content')
+    {{-- @todo make this work in dark mode --}}
```
Comment (Leping08) [inline resources/views/pages/trails/legacy.blade.php]: This page looks like it works in dark mode. If so can you remove this comment?

**Review by Leping08 — CHANGES_REQUESTED** [review]

File: resources/views/pages/trails/legacy.blade.php (hunk 1 above)
Comment (Big-nuche) [reply resources/views/pages/trails/legacy.blade.php]: Updated

File: resources/views/pages/trails/legacy.blade.php (hunk 2 above)
Comment (Big-nuche) [reply resources/views/pages/trails/legacy.blade.php:5]: Updated

File: resources/views/pages/trails/legacy.blade.php (hunk 3 above)
Comment (Big-nuche) [reply resources/views/pages/trails/legacy.blade.php]: Updated

**Pull request merged**